### ISSUE TITLE
Companion crashes when try to edit radio settings (N443/N444) 

### ISSUE THREAD

**Issue by x10user** [opentx/opentx#5425]
since N443/N444 Companion crashes when try to edit radio settings (for X10)

This worked fine in older nightly builds.

![image](https://user-images.githubusercontent.com/33264007/33232459-2716d354-d207-11e7-9ee7-a78557383f0e.png)

**Comment by kilrah** [comment]
Details please... blank model? Already some data? If yes supply the file

**Comment by x10user** [comment]
Observed with active TX-Type Horus X-10 but I can reproduce it also with X9D+.

Further information (don't know if valuable): Windows 10, Companion application language is English.
What kind of file helps?

**Comment by 3djc** [comment]
We need a reliable way to replicate the problem

**Comment by kilrah** [comment]
The otx file that causes the crash if any, or instructions to reproduce the problem from a blank configuration.

**Comment by x10user** [comment]
[FrSky X10.zip](https://github.com/opentx/opentx/files/1504178/FrSky.X10.zip)
I attached the otx-file for reference.

It does not crash if I start from an empty otx-file.
I was wondering if it might been caused by the assignment of trim buttons to the adjust GV function from the special functions in the TX settings. These are enabled (at least were enaled in earlier nightly builds) but not available at the system menu at the transmitter. This is an inconsistency between Companion and transmitter firmware.
Other than that I set up some telemetry screens and sensors at the Transmitter whereas these options are currently not available at Companion.
